src/website/catalog/migrations/0001_initial.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.1.7 on 2023-07-24 21:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Regcomp',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(blank=True, max_length=255, null=True, unique=True)),
+                ('note1', models.TextField(blank=True, null=True)),
+                ('note2', models.TextField(blank=True, null=True)),
+                ('city', models.CharField(blank=True, max_length=255, null=True)),
+                ('website', models.TextField(blank=True, null=True)),
```

Comment:
почему эти field'ы могут быть пустыми или null? вроде такое не должно быть возможно
```suggestion
                ('name', models.CharField(blank=False, max_length=255, null=False, unique=True)),
                ('note1', models.TextField(blank=False, null=False)),
                ('note2', models.TextField(blank=False, null=False)),
                ('city', models.CharField(blank=False, max_length=255, null=False)),
                ('website', models.TextField(blank=False, null=False)),
```

Reply:
проверила с этими изменениями, все отработало корректно

Reply:
а как мы будем скрапить данные с тех провайдеров, у которых нет website?

Reply:
на встрече решили, что везде должно быть False